### ISSUE TITLE
request:  The lifetime of deployments is not clear to me

### ISSUE THREAD

**Issue by abdallahshaban557** [invertase/globe#18]
### Is there an existing feature request for this?

- [X] I have searched the existing issues.

### Description

As a developer, when I deploy a version of my backend it is not clear to me what the lifetime of that deployment is, and whether I can delete the old deployments from having to show up in my console.

### Reasoning

I think it would make sense to have only a handful of environments that are active at any point in time. And I cannot see a way to understand (from the console or from docs) on what the lifetime of my deployments are. Will the links created by Globe continue to be alive at all times?

**Comment by Ehesp** [comment]
All deployments exist forever. Each deployment has its own uniquely generated URL, which you could use to test against for example or verify changes are working before pushing to production.

Deployments marked as production still have a unique URL, but also get assigned the project domain(s) when successfully deployed.

Its possible for us to prevent deployments from being accessible after a certain time frame, but this isn't implemented.

**Comment by abdallahshaban557** [comment]
I believe that from a DevOps and security perspective, it would make sense that a developer can turn off access to and completely delete previous deployments. What happens if I have a vulnerability for example that was previously deployed that I want to delete? Would you also charging in the future for these environments existing (Especially if I have data or other things as part of my deployment)?

**Comment by Ehesp** [comment]
Makes sense, we can quite easily delete those deployments - just not possible at the moment (will make a feature request).

Could you clarify on what you mean by "data" in the deployment here:

> Especially if I have data or other things as part of my deployment

There will be no change keeping old deployments around on our side, it's part of the architecture decision we made and built.

**Comment by alestiago** [comment]
I am interested in the "easily delete those deployments" feature as well.  Is there an option to delete a project entirely (not just a deployment)?

**Comment by Ehesp** [comment]
Thanks for the feedback, once we complete the road map it'll give more insight into what we're working on.

Yeah project and account will be implemented - deleting a project is partly done but needs more tests on our side.

**Comment by kaziwaseef** [comment]
Project Deletion has been implemented.
Our Releases and Roadmaps can be found here: https://docs.globe.dev/releases/roadmap
